lib/perl/Genome/Qc/Tool/VerifyBamId.pm

```diff
@@ -56,4 +50,12 @@ sub qc_metrics_file_accessor {
     return 'out_prefix';
 }
 
+sub _non_metric_columns {
+    return qw(#SEQ_ID RG);
```

Comment:
This will result in 'Possible attempt to put comments in qw() list' messages.  You could locally turn off warnings from qw or build the array without qw().  http://www.perlmonks.org/index.pl/?node=Using%20%23%20inside%20qw%28%29